2-advanced/dubbo-samples-service-discovery/dubbo-servicediscovery-migration/dubbo-servicediscovery-migration-provider2/src/main/resources/dubbo.properties

```diff
@@ -1,2 +0,0 @@
-dubbo.application.qos.port=22222
-dubbo.application.register-mode=instance
```

Comment:
`dubbo.application.register-mode=instance` is missed in `application.yml`

Reply:
Have been modified，please review